index.html

```diff
@@ -26,88 +26,7 @@
 
         <link href="https://fonts.googleapis.com/css?family=Source+Code+Pro" rel="stylesheet">
-        <link rel="stylesheet" href="./dist/css/semantic.min.css">
-        <style>
-            body {
-                background: -webkit-linear-gradient(left bottom, #005C97, #363795); /* For Safari 5.1 to 6.0 */
-                background: -o-linear-gradient(top right, #005C97, #363795); /* For Opera 11.1 to 12.0 */
-                background: -moz-linear-gradient(top right, #005C97, #363795); /* For Firefox 3.6 to 15 */
-                background: linear-gradient(to top right, #005C97, #363795); /* Standard syntax */
-            }
-            .holder {
-                height: inherit;
-                max-width: 300px;
-            }
-            .holder .middle {
-                height: 100%;
-            }
-            .holder .middle .segment {
-                display: flex;
-                flex-direction: column;
-                min-height: 300px;
-                padding: 35px;
-            }
-            .holder .ui.error.message {
-                margin: 0px 0px 10px 0px;
-            }
-            .logo img {
-                display: block;
-                margin: auto;
-                width: 50%;
-                margin-bottom: 20px;
-            }
-            .hexData {
-                font-family: 'Source Code Pro', monospace;
-                font-weight: 600;
-                font-size: 16px;
-                background: grey;
-                color: white;
-                padding: 5px;
-                min-height: 30px;
-                cursor: pointer;
-            }
-            #snackbar {
-                visibility: hidden;
-                min-width: 250px;
-                margin-left: -125px;
-                background-color: #333;
-                color: #fff;
-                text-align: center;
-                border-radius: 2px;
-                padding: 16px;
-                position: fixed;
-                z-index: 1;
-                left: 50%;
-                bottom: 30px;
-                font-size: 17px;
-            }
-
-            #snackbar.show {
-                visibility: visible;
-                -webkit-animation: fadein 0.5s, fadeout 0.5s 2.5s;
-                animation: fadein 0.5s, fadeout 0.5s 2.5s;
-            }
-
-            @-webkit-keyframes fadein {
-                from {bottom: 0; opacity: 0;}
-                to {bottom: 30px; opacity: 1;}
-            }
-
-            @keyframes fadein {
-                from {bottom: 0; opacity: 0;}
-                to {bottom: 30px; opacity: 1;}
-            }
-
-            @-webkit-keyframes fadeout {
-                from {bottom: 30px; opacity: 1;}
-                to {bottom: 0; opacity: 0;}
-            }
-
-            @keyframes fadeout {
-                from {bottom: 30px; opacity: 1;}
-                to {bottom: 0; opacity: 0;}
-            }
-        </style>
+        <link rel="stylesheet" href="./dist/style.css">
```

Comment:
The file name should contain `.min` suffix as it is minified. Please do the same for js file as well.

Reply:
Will amend.

There won't be an *unminified* version of the files so the suffix is redundant in this case.

Reply:
This will help new contributors distinguish between `styles` in dist and `src/css`.